tensorboard/examples/plugins/example_raw_scalars/test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for the example plugin."""
+
+import unittest
+
+from werkzeug import test
+from werkzeug import wrappers
+
+from tensorboard.plugins import base_plugin
+from tensorboard_plugin_example_raw_scalars import plugin
+
+example_plugin = plugin.ExampleRawScalarsPlugin(base_plugin.TBContext())
+serve_static_file = example_plugin._serve_static_file
```

Comment:
Avoid executing code at module import time. Failures at module import
time are basically impossible to recover from. This should run inside
the relevant test method, either explicitly (inlined or as a function
call) or implicitly (via `setUp`).

Reply:
Done.